playbooks/openstack/configuration.md

```diff
@@ -568,7 +568,7 @@ the `build_image` playbook.
 If you don't want to be setting the build variables in your inventory, you can
 pass them to ansible-playbook directly:
 
-    ansible-playbook -i inventory openshift-ansible/playbooks/openstack/openshift-cluster/build_image.yml -e openshift_openstack_build_base_image=CentOS-7-x86_64-GenericCloud-1805 -e openshift_openstack_build_network_cidr=192.168.42.0/24
+    ansible-playbook -i inventory openshift-ansible/playbooks/openstack/openshift-cluster/build_image.yml -e openshift_openstack_build_base_image=CentOS-7-x86_64-GenericCloud-1805 -e openshift_openstack_build_network_cidr=192.168.42.0/24 -e 'openshift_version="3.10"'
```

Comment:
You definitely don't want to set openshift_version with extra_vars, nor any other version related variables such as openshift_release